test/json/generic-conf.json

```diff
@@ -0,0 +1,40 @@
+{
+    "instanceServiceSpec": {
+        "ports": [
+            {
+                "name": "http",
+                "port": 6052,
+                "protocol": "TCP",
+                "targetPort": 6052
+            }
+        ],
+        "type": "ClusterIP"
+    },
+    "brokerPodSpec": {
+        "containers": [
+            {
+                "image": "nginx:latest",
+                "name": "broker"
+            }
+        ]
+    },
+    "capacity": 5,
+    "configurationServiceSpec": {
+        "ports": [
+            {
+                "name": "http",
+                "port": 6052,
+                "protocol": "TCP",
+                "targetPort": 6052
+            }
+        ],
+        "type": "ClusterIP"
+    },
+    "discoveryHandler": {
+        "name": "opcua",
+        "discoveryDetails" : {
+            "discoveryHandlerConfig": "\"opcua\":{\"opcuaDiscoveryMethod\":{\"standard\":{\"discoveryUrls\": [\"opc.tcp://127.0.0.1:4855/\"]}}, \"applicationNames\": { \"action\": \"Exclude\", \"items\": [\"Some application name\"]}}"
```

Comment:
i see "opcua" twice, does this need to be updated still?

Reply:
this was an unused test file. removed in 8698b3197b9